python_modules/dagster/dagster/_core/storage/runs/schema.py

```diff
@@ -105,6 +105,14 @@
 
 db.Index("idx_run_tags", RunTagsTable.c.key, RunTagsTable.c.value, mysql_length=64)
 db.Index("idx_run_partitions", RunsTable.c.partition_set, RunsTable.c.partition, mysql_length=64)
+db.Index(
+    "idx_runs_by_job",
+    RunsTable.c.pipeline_name,
+    RunsTable.c.id,
+    mysql_length={
+        "pipeline_name": 255,
```

Comment:
confident in this 255? align with similar settings?

Reply:
I'll switch this to 512... that's what we use in our mysql compatibility code to coerce text fields in other places.